jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRoleItemView.java

```diff
@@ -16,84 +16,266 @@
 
+import com.google.gwt.user.client.TakesValue;
 import org.jboss.errai.common.client.api.IsElement;
+import org.jboss.errai.common.client.dom.Button;
 import org.jboss.errai.common.client.dom.Div;
 import org.jboss.errai.common.client.dom.HTMLElement;
 import org.jboss.errai.common.client.dom.Span;
 import org.jboss.errai.common.client.dom.UnorderedList;
+import org.jboss.errai.databinding.client.api.DataBinder;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jboss.errai.ui.shared.api.annotations.AutoBound;
+import org.jboss.errai.ui.shared.api.annotations.Bound;
 import org.jboss.errai.ui.shared.api.annotations.DataField;
 import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.cm.client.events.CaseRoleAssignmentListLoadEvent;
+import org.jbpm.workbench.cm.client.events.CaseRoleAssignmentListOpenEvent;
+import org.jbpm.workbench.cm.client.roles.util.ItemsLine;
+import org.jbpm.workbench.cm.client.util.AbstractView;
+import org.jbpm.workbench.cm.client.util.ConfirmPopup;
+import org.jbpm.workbench.cm.model.CaseRoleAssignmentSummary;
 
 import static org.jboss.errai.common.client.dom.DOMUtil.addCSSClass;
 import static org.jboss.errai.common.client.dom.DOMUtil.removeCSSClass;
 import static org.jboss.errai.common.client.dom.Window.getDocument;
+import static org.jbpm.workbench.cm.client.resources.i18n.Constants.*;
 
 @Dependent
-@Templated(stylesheet = "CaseRoleItemView.css")
-public class CaseRoleItemView implements IsElement {
+@Templated
+public class CaseRoleItemView extends AbstractView<CaseRolesPresenter> implements TakesValue<CaseRoleAssignmentSummary>, IsElement {
+    public static String USERS_LINE_ID="_users";
+    public static String GROUP_LINE_ID = "_groups";
 
     @Inject
-    @DataField("role-name")
-    Span roleName;
+    @DataField("list-group-item")
+    Div listGroupItem;
+
+    @Inject
+    @DataField("actions-dropdown")
+    Div actions;
 
     @Inject
-    @DataField("name")
+    @DataField("actions-button")
+    Button actionsButton;
+
+    @Inject
+    @DataField("actions-items")
+    UnorderedList actionsItems;
+
+    @Inject
+    @DataField("role-name")
+    @Bound
     Span name;
 
     @Inject
-    @DataField("icon-type")
-    Span iconType;
+    @DataField("line-users")
+    ItemsLine usersItemsLine;
 
     @Inject
-    @DataField("list-group-item")
-    Div listGroupItem;
+    @DataField("div-users")
+    Div usersDiv;
 
     @Inject
-    @DataField("actions-dropdown")
-    Div actions;
+    @DataField("line-groups")
+    ItemsLine groupsItemsLine;
 
     @Inject
-    @DataField("actions-items")
-    UnorderedList actionsItems;
+    @DataField("div-groups")
+    Div groupsDiv;
+
+    @Inject
+    ConfirmPopup confirmPopup;
+
+    @Inject
+    protected TranslationService translationService;
+
+    @Inject
+    @AutoBound
+    private DataBinder<CaseRoleAssignmentSummary> caseRoleAssignmentSummary;
+
+    boolean listenLoadLineEvent=true;
```

Comment:
If this is only set to false when the instance is destroyed, is it needed at all?